composer.json

```diff
@@ -4,83 +4,100 @@
     "description": "The SURFnet Step-up Middleware exposes an API for managing Step-up Identities and Tokens.",
     "autoload": {
         "psr-4": {
-            "Surfnet\\": "src\\Surfnet"
+            "Surfnet\\Stepup\\": "src\\Surfnet\\Stepup",
+            "Surfnet\\StepupMiddleware\\": "src\\Surfnet\\StepupMiddleware"
         },
         "classmap": [
-          "app/AppKernel.php",
-          "app/AppCache.php"
+            "src/Kernel.php"
         ]
     },
     "autoload-dev": {
-        "classmap": ["src/Surfnet/StepupMiddleware/ApiBundle/Tests/Request/commands.php"]
+        "classmap": [
+            "src/Surfnet/StepupMiddleware/ApiBundle/Tests/Request/commands.php"
+        ]
     },
     "minimum-stability": "stable",
     "require": {
-        "php": "^5.6|^7.0",
+        "php": "^7.1",
         "ext-gmp": "*",
-        "symfony/symfony": "3.4.*",
-        "symfony/swiftmailer-bundle": "~2.3",
-        "symfony/monolog-bundle": "~2.4",
-        "twig/twig": "^1.35",
-        "twig/extensions": "~1.5",
-        "doctrine/cache": "~1.4",
-        "doctrine/common": "~2.5",
-        "doctrine/dbal": "~2.5",
-        "doctrine/doctrine-bundle": "~1.4",
-        "doctrine/doctrine-migrations-bundle": "^1.1",
-        "doctrine/migrations": "~1.4",
-        "doctrine/orm": "~2.5",
-        "sensio/distribution-bundle": "~5.0",
-        "sensio/framework-extra-bundle": "~3.0",
+        "ext-json": "*",
```

Comment:
`ext-openssl` and `ext-pdo` also are required extensions. Please add those to the `require` section

Reply:
It's of out of scope for the upgrade when I'm touching the code when working on other features I'll boyscout this.